public/app/plugins/panel/geomap/editor/MapViewEditor.tsx

```diff
@@ -94,9 +130,69 @@ export const MapViewEditor: FC<
           </InlineFieldRow>
         </>
       )}
+      {value?.id === MapCenterID.Fit && (
```

Comment:
It might be nice to break up this component up into separately defined fragments for better readability / maintainability (not just additions here but the entire component)